manifest.php

```diff
@@ -43,7 +43,7 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '25.10.2',
+    'version'     => '25.10.3',
     'author'      => 'Open Assessment Technologies',
     'requires' => [
         'taoItems' => '>=10.11.0',
```

Comment:
Not really required, but can be bumped.
```suggestion
        'taoItems' => '>=10.13.0',
```